catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -209,6 +214,9 @@ QueryResponse query(
       boolean overrideFanoutRename,
       boolean fanoutEnabled)
       throws UnsupportedQueryException, FederationException {
+    DateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
+    format.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
Should the date/time match the log4j timestamp format?  It looks to me like log4j outputs time using local time without a timezone, eg 

`2012-11-02T14:34:02,781`